index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inspiration&family=Nanum+Gothic:wght@400;700;800&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="css/style.css">
+  <title>Albert Einstein Tribute Page</title>
+</head>
```

Comment:
**Webhint linter error:** _"A 'viewport' meta element was not specified."_
Please include a `<meta>` tag in the  `<head>` of the HTML, this gives the browser instructions on how to control the page's dimensions and scaling.
Example: `<meta name="viewport" content="width=device-width, initial-scale=1.0">`